TestDockerfile39

```diff
@@ -1,13 +1,28 @@
 FROM python:3.9-slim
 WORKDIR /app
-RUN apt-get update -y && apt-get install -y python3-dev libcurl4-openssl-dev libgnutls28-dev libjpeg-progs libimage-exiftool-perl gifsicle scons python3-all-dev libboost-python-dev libexiv2-dev ffmpeg make zlib1g-dev gcc libssl-dev libjpeg-dev libwebp-dev redis && apt-get clean
-RUN apt-get update -y && apt-get install --reinstall -y build-essential && apt-get clean
-RUN pip install --upgrade pip
-COPY setup.py /app/setup.py
-RUN mkdir -p /app/thumbor
-RUN touch /app/thumbor/__init__.py
+RUN apt-get update -y && \
+    apt-get install -y \
```

Comment:
Adding `–no-install-recommends` and  `–no-install-suggests` will avoid unnecessary packages